<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TSPGenMainWindow</class>
 <widget class="QMainWindow" name="TSPGenMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TSPGen</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsViewMap"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBoxDisplayCities">
        <property name="text">
         <string>Display cities</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxDisplayConnections">
        <property name="text">
         <string>Display connections</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxDisplayBestUnit">
        <property name="text">
         <string>Display best unit</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="actionImportMap"/>
    <addaction name="actionGenerateMap"/>
   </widget>
   <widget class="QMenu" name="menuPopulation">
    <property name="title">
     <string>Population</string>
    </property>
    <addaction name="actionGeneratePopulation"/>
    <addaction name="actionDisplayPopulation"/>
   </widget>
   <widget class="QMenu" name="menuEvolution">
    <property name="title">
     <string>Evolution</string>
    </property>
    <addaction name="actionSetParameters"/>
    <addaction name="actionProcessGeneration"/>
   </widget>
   <addaction name="menuMap"/>
   <addaction name="menuPopulation"/>
   <addaction name="menuEvolution"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionImportMap">
   <property name="text">
    <string>Import...</string>
   </property>
  </action>
  <action name="actionGenerateMap">
   <property name="text">
    <string>Generate</string>
   </property>
  </action>
  <action name="actionGeneratePopulation">
   <property name="text">
    <string>Generate new...</string>
   </property>
  </action>
  <action name="actionDisplayPopulation">
   <property name="text">
    <string>Display</string>
   </property>
  </action>
  <action name="actionProcessGeneration">
   <property name="text">
    <string>Process generation</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionSetParameters">
   <property name="text">
    <string>Set parameters...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
